literature.bib

```diff
@@ -28309,3 +28450,80 @@ @misc{missingdatatype
 	year = 2024,
 	howpublished = {\url{https://www.hillelwayne.com/post/graph-types}}
 }
+
+@book{Triola2018,
+    title={{Elementary statistics}},
+  author={Triola, Mario F},
+  year = 2018,
+  publisher = {Pearson},
+}
+
+@misc{GNU:patch,
+  author = {Wall, Larry and Eggert, Paul and Davison, Wayne and MacKenzie, David and Grünbacher, Andreas},
+  title = {{GNU patch}},
+  year = 2009,
+  howpublished = {Website: \url{https://savannah.gnu.org/projects/patch/}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{GNU,
+  author = {Stallman, Richard},
+  title = {{The GNU project}},
+  year = 2023,
+  howpublished = {Website: \url{https://www.gnu.org/gnu/thegnuproject.html}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{Linux:ApplyingPatches,
+  author = {Juhl, Jesper},
+  title = {{Applying Patches To The Linux Kernel}},
+  year = 2016,
+  howpublished = {Website: \url{https://www.kernel.org/doc/html/v4.11/process/applying-patches.html}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{Linux:SubmittingPatches,
+  author = {The kernel development community},
+  title = {{Submitting patches: the essential guide to getting your code into the kernel}},
+  year = 2024,
+  howpublished = {Website: \url{https://www.kernel.org/doc/html/latest/process/submitting-patches.html}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{Git:Apply,
+  author = {Linus Torvalds and Junio C Hamano and others},
+  title = {{git-apply}},
+  year = 2023,
+  publisher = {{Software Freedom Conservancy}},
+  howpublished = {Website: \url{https://git-scm.com/docs/git-apply}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{Git:CP,
+  author = {Linus Torvalds and Junio C Hamano and others},
+  title = {{git-cherry-pick}},
+  year = 2023,
+  publisher = {{Software Freedom Conservancy}},
+  howpublished = {Website: \url{https://git-scm.com/docs/git-cherry-pick}},
+  note = {Accessed: 2024-07-01},
+}
+
+@misc{BusyBox,
+    title={{Busybox: A Swiss Army Knife for Linux}},
+	author={Wells, Nicholas},
+	journal={Linux Journal},
+	volume={2000},
+	number={78es},
```

Comment:
If this is available, maybe this is not a misc entry but an article?

Reply:
It seems that the article was published within a magazine that is now discontinued.  The only available version of this article can now be found under [https://dl.acm.org/doi/fullHtml/10.5555/364412.364422](https://dl.acm.org/doi/fullHtml/10.5555/364412.364422). 

I could update the entry to point to this doi instead. What do you think?

Reply:
Yes, adding the doi is good
Still I think I would categorize it as an article